runtime/runtime-params-estimator/src/main.rs

```diff
@@ -117,7 +118,7 @@ fn main() -> anyhow::Result<()> {
             )
             .expect("failed to init config");
 
-            let near_config = load_config(&state_dump_path);
+            let near_config = load_config(&state_dump_path, GenesisValidationMode::Full);
```

Comment:
Do we really need to validate genesis in parameter estimator? @matklad

Reply:
I don't think it matters either way -- I'd expect the performance to be dominated by other stuff, and an extra sanity check won't harm.